### PR TITLE
MTV 839 Enabling deployment on OKE

### PR DESCRIPTION
**Pull request by CarmiWisemon**
MTV Version 2.5.4

Resolves - https://issues.redhat.com/browse/MTV-839

Deploy preview - file:///home/cwisemon/repos/forklift-documentation/documentation/doc-Migration_Toolkit_for_Virtualization/build/tmp/en-US/html-single/index.html#compatibility-guidelines_mtv

### CODE REVIEW

File: documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc [hunk 1]
```diff
@@ -36,6 +36,11 @@ include::modules/snippet_ova_tech_preview.adoc[]
 Migration using {osp} source providers only supports VMs that use only Cinder volumes.
 ====
 
+[NOTE]
+====
+Deployment of MTV on OpenShift Kubernetes Engine (OKE) is enabled. For more information, see link:https://docs.openshift.com/container-platform/4.14/welcome/oke_about.html[About OpenShift Kubernetes Engine].
+====
```
Comment (ahadas) [inline documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc:42]: the content of the note looks good but wouldn't it make more sense to place it in the `Software compatibility guidelines` section?

Comment (ahadas) [reply documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc:42]: @CarmiWisemon maybe not 'as is' - we can also say that it was added in version 2.5.3 (it's not possible to deploy earlier versions of MTV 2.5 on OKE), but yes - to that section please

Comment (ahadas) [reply documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc:42]: the other note looks good - no need for this one now

Comment (CarmiWisemon) [reply documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc:42]: @ahadas 
I added the updated note in the "Software compatibility guidelines" section.
Can we also keep the note in the "About..." section?
Thanks,
Carmi

Comment (ahadas) [reply documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc:42]: > Can we also keep the note in the "About..." section?

I don't think we need to keep the note in the "About..." section, deployment of OKE is not that significant - I'd say that writing about it in the software compatibility guidelines is enough

Comment (CarmiWisemon) [reply documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc:42]: Will do!

**Review by ahadas — APPROVED** [review]

**Review by qiyuann — APPROVED** [review]

**Pull request merged**